lib/network/modules/components/Node.js

```diff
@@ -139,10 +140,18 @@ class Node {
 
     // this transforms all shorthands into fully defined options
     Node.parseOptions(this.options, options, true, this.globalOptions, this.grouplist);
-
-    if (options.opacity !== undefined) {
+    
+    
+    if (options.opacity !== undefined && Node.checkOpacity(options.opacity)) {
+      if (this.id >= 4 && this.id <= 7) {
+        console.log(options.opacity);
+      }
+      // FIXME: This doesn't work if a group has already set this option
```

Comment:
I need some help with the options, I can't seem to find why the group opacity overrides the setting of the opacity here

Reply:
Fixed it